spec/presenters/spelling_suggestion_presenter_spec.rb

```diff
@@ -19,9 +19,9 @@
             dimension81: "full english",
           },
         },
-        keywords: "full english",
-        highlighted: "<mark>full</mark> english",
-        link: "/breakfast-finder?keywords=full+english" }]
+                    keywords: "full english",
+                    highlighted: "<mark>full</mark> english",
+                    link: "/breakfast-finder?keywords=full+english" }]
```

Comment:
odd indentation adjustment here

Reply:
Yeah, I noticed that too. It's because we (for no apparent reason) turned off [Layout/FirstHashElementIndentation](https://github.com/alphagov/rubocop-govuk/blob/master/config/layout.yml#L43). Well done us.

My thinking is that someone can easily fix this manually, if they want to. I'm leaving as-is deliberately, in favour of switching on that Cop to do the work for me.